doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
I tried to get things into a consistent style across the docs, and part of that is to use code formatting for code and removing the code formatting from file names and paths that are not code. It provides a better visual distinction IMO, and that's more common in other parts of the docs.